src/Nordea/Components/Button.elm

```diff
@@ -108,13 +112,22 @@ withStyles styles (Button config) =
 
 view : List (Attribute msg) -> List (Html msg) -> Button -> Html msg
 view attributes children (Button config) =
+    let
+        variantSpecificChildren =
+            case config.variant of
+                Card ->
+                    [ Icons.arrowRight [ css [ width (rem 1), height (rem 1) ] ] ]
+
+                _ ->
+                    [ Html.nothing ]
+    in
     Html.styled Html.button
```

Comment:
The front page of PH needs these card buttons as a link instead of a button to be semantically correct. Based on the answer [here](https://stackoverflow.com/questions/6393827/can-i-nest-a-button-element-inside-an-a-using-html5), I see that we cannot simply wrap these buttons in a `a`-tag where needed, as that is not valid html.

I think we need a way to change the tag dynamically. Maybe we can do it the same way we do it for the typography?